mesh/pmesh.cpp

```diff
@@ -4823,6 +4824,7 @@ void ParMesh::PrintAsOne(std::ostream &out) const
          {
             // create a wrapper ParGridFunction
             ParGridFunction ParNodes(pfes, Nodes);
+            ParNodes.HostRead();
             ParNodes.SaveAsOne(out);
```

Comment:
`HostRead` is unnecessary, `SaveAsOne` already does that.

Reply:
`done`